Rakefile

```diff
@@ -0,0 +1,52 @@
+namespace :db do
+  desc "Run migrations"
+  task :migrate, [:version] do |t, args|
+    require "sequel"
+    Sequel.extension :migration
+    if (ENV['RACK_ENV'] == "test")
```

Comment:
Added if statement that checks environment so `RACK_ENV=test rake db:migrate` will work